apps/site/assets/ts/stop/__tests__/sidebar/__snapshots__/ParkingTest.tsx.snap

```diff
@@ -303,6 +308,11 @@ exports[`Parking via enzyme-to-json it displays parking info when opened and inn
                 </strong>
                 4
               </li>
+              <li>
```

Comment:
You could add to the parking data structure in `ParkingTest.tsx` so that it outputs data in the snapshot.